internal/provisioner/bundledeployment/bundledeployment.go

```diff
@@ -426,7 +450,13 @@ func (p *bundledeploymentProvisioner) getReleaseState(cl helmclient.ActionInterf
 	desiredRelease, err := cl.Upgrade(obj.GetName(), p.releaseNamespace, chrt, values, func(upgrade *action.Upgrade) error {
 		upgrade.DryRun = true
 		return nil
-	})
+	},
+		// To be refactored issue 534
```

Comment:
nit: can we copy/paste the 534 issue link so it's easier to track when reading through the codebase locally?

Reply:
Sure.  It must be better.